src/components/dialogs/two-windings-transformer/phase-tap-changer-pane.js

```diff
@@ -520,7 +520,10 @@ const PhaseTapChangerPane = (props) => {
                                 onClick={() => generateTapRows()}
                                 disabled={
                                     !phaseTapChangerEnabled ||
-                                    !(lowTapPosition && highTapPosition)
+                                    !(
```

Comment:
Could simplify to `lowTapPosition === '' || highTapPosition === ''`, that would avoid double negation.

Reply:
you're right
Done